packages/resolver-hoogle-search/__tests__/index.js

```diff
@@ -18,8 +18,8 @@ describe('hoogle-search', () => {
     assert.deepEqual(typeof hoogleSearch({ target }), 'function');
   });
 
-  it('calls the hoogle search api', () => {
-    hoogleSearch({ target })();
+  it('calls the hoogle search api', async () => {
+    await hoogleSearch({ target })();
```

Comment:
`hoogleSearch` returns a promise but since this test isn't setting `response = []` (fixed in the other file) it was crashing with no real information, by awaiting it jest reports the failure correctly.